ceph.spec.in

```diff
@@ -277,6 +279,8 @@ Requires(pre):	pwdutils
 %if (0%{?rhel} && 0%{?rhel} <= 6)
 Requires:	python-argparse
 %endif
+Conflicts:      ceph < %{epoch}:10
+Obsoletes:      ceph < %{epoch}:10
```

Comment:
Hmm, did we put any of the files from the ceph package to ceph-common in the package split?